src/Matcher/UlidMatcher.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Coduo\PHPMatcher\Matcher;
+
+use Coduo\PHPMatcher\Backtrace;
+use Coduo\PHPMatcher\Parser;
+use Coduo\ToString\StringConverter;
+use Symfony\Component\Validator\Constraints\Ulid;
+
+final class UlidMatcher extends Matcher
+{
+    /**
+     * @var string
+     */
+    public const PATTERN = 'ulid';
+
+    /**
+     * @var string
+     */
+    public const ULID_PATTERN = '[0123456789ABCDEFGHJKMNPQRSTVWXYZabcdefghjkmnpqrstvwxyz]{26}';
```

Comment:
hey, did you maybe took this pattern from somewhere? If yes, could you please share the source?
Thanks for the contribution!

Reply:
Hey ! 

Indeed, I've been inspired by the Symfony ULID validation method

https://github.com/symfony/uid/blob/8311a3f6e14c21960e7955452fe52a462d58ad2b/Ulid.php#L41-L52

Reply:
Would you mind to put that link in a comment to this patrern and mention Symfony uid lib as an inspiration?